sdk/python/feast/type_map.py

```diff
@@ -284,6 +283,21 @@ def _python_value_to_proto_value(feast_value_type: ValueType, value: Any) -> Pro
         if value is None:
             return ProtoValue()
 
+        if feast_value_type == ValueType.UNIX_TIMESTAMP_LIST:
```

Comment:
I can't see how I'd do that? This is a special case as complex type conversions are needed. This is already done for the non-list case: https://github.com/feast-dev/feast/blob/c84a6fc7e6cff4d87a8324a52bc37ac299f977e7/sdk/python/feast/type_map.py#L321

I've essentially replicated this but for the LIST case.

Reply:
I've now removed `UNIX_TIMESTAMP_LIST` from `PYTHON_LIST_VALUE_TYPE_TO_PROTO_VALUE` to reflect this.

Reply:
fyi I reverted this change because it broke some tests - having UNIX_TIMESTAMP_LIST in PYTHON_LIST_VALUE_TYPE_TO_PROTO_VALUE was necessary. Also I understand why my request was wrong, so please ignore it.